docs/contributing/contributor-guidelines.md

```diff
@@ -1,3 +1,31 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Question.svg" /> Contributor Guidelines
 
 ---
+
+Decred development is, like the project, decentralized. Different projects and their repos can have their own own rules and norms. However, the general process for contributing across repos will be the same. Below we describe the basic process for finding work, engaging projects and the general GitHub process.
+
+## Finding Work
+
+Nearly all work occurs in public repos and is open to new contributors. A full list of repos can be found in the Decred [GitHub org](https://github.com/decred). A list of projects under active development and descriptions of them can be found on the [Projects](../../projects) page. 
+
+If new to the project, be sure to check out the user-facing [docs](https://docs.decred.org) to get an understanding of the basics of Decred. Then find a project that interests you and that you feel you have the skills to contribute to. Once you've found a project you're interested in, find an issue you'd like to work on. GitHub Issues tagged with `good first issue` are a good place to start. 
+
+## Contributing
+
+Once you've found an issue you'd like to work on, announce your plans **before** starting work. This avoids duplicate work, minimizes the chances you're working on something that might not fit with the consensus of the community, and permits discussions that can help you achieve your goal. Most projects have a dedicated chat channel for discussing project-specific work. 
```

Comment:
We should explain **how** to do the announcement. Typically commenting on the issue or by speaking with relevant devs on Matrix.

Reply:
Agreed. Reworded to:

>Once you've found something you'd like to work on, announce your plans **before** starting work.  This avoids duplicate work, minimizes the chances you're working on something that might not fit with the consensus of the community, and permits discussions that can help you achieve your goal. If working on a GitHub Issue, you can do this by commenting on the Issue. You can also speak with the relevent devs on Matrix. Most projects have a dedicated chat channel for discussing project-specific work.